zubhub_frontend/zubhub/src/components/Sidenav/data.js

```diff
@@ -17,7 +17,7 @@ export const links = ({ draftCount, myProjectCount, auth, t }) => [
     { label: t('pageWrapper.sidebar.projects'), link: '/', icon: Dashboard },
     { label: t('pageWrapper.sidebar.profile'), link: '/profile', icon: Person, reactIcon: true, requireAuth: true },
     { label: t('pageWrapper.sidebar.createProject'), link: '/projects/create', icon: EmojiObjects },
-    ...(auth?.tags.includes('staff')
+    ...(['staff', 'creator'].some(tag => auth?.tags.includes(tag))
```

Comment:
creators can see the `Create Activity` link in the SideNav

Reply:
@coderatomy Can we revert this? We don't want creators to see create activities for now. Only "staff" and "educator" tags can create activities

Reply:
I was referring from your comment here. It kind of confused things here. https://github.com/unstructuredstudio/zubhub/pull/885#discussion_r1367418866

Reply:
I think let me just add the `educator` here too?

Reply:
Yes, we can add `educator` tag. Only staff and educator roles can create activities for now and hence only they should be able to see "Create Activity" link in sidenav. Added some more clarification here: https://github.com/unstructuredstudio/zubhub/pull/885#discussion_r1372012535

Reply:
Done